### DISCUSSION TITLE
Does MSIX / AppInstaller allow custom release channels for updates?

### DISCUSSION THREAD

**Discussion by MarkIngramUK** [microsoft/WindowsAppSDK#1316] [Q&A] [answered]
As per the title, is it possible to allow an end user to choose between different channels of a release? e.g. Alpha / Beta / Retail. Or is it a case of installing the same app N times, via a different MSIX / `.appinstaller`?

**Comment by btueffers** [comment, marked as answer]
Hi @MarkIngramUK, you must compile your app with the specific channel you want to use. We have no current plans to support otherwise. Thanks for your feedback

**Reply by MarkIngramUK** [reply]
Thanks @btueffers , I wasn't sure whether I could use the `PackageManager` APIs to change the URL of the `.appinstaller` file.
